### PR TITLE
feat: splash screen

### PR DESCRIPTION
**Pull request by Jaszkowic**

### CODE REVIEW

File: src/components/splash/splash.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+				<div className="flex flex-row w-full justify-between items-top pb-8">
+					<div className="flex flex-col gap-2 pr-6 w-full lg:max-w-[70%]">
+						<Markdown className="xl:text-2xl 2xl:text-3xl font-semibold">
+							{i18n.splash.headline}
+						</Markdown>
+						<div className="text-sm xl:text-base">
+							{i18n.splash.subheadline}
+						</div>
+						<div>
+							<PrimaryButton label={"Los geht's"} onClick={onClose} />
```
Comment (aeschi) [inline src/components/splash/splash.tsx:80]: internationalization missing

File: src/components/splash/splash.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	icon: React.ReactNode;
+}
+
+const SectionHeading: React.FC<SectionHeadingProps> = ({
+	icon,
+	title,
+	content,
+}) => {
+	return (
+		<div className="flex flex-col gap-2">
```
Comment (aeschi) [inline src/components/splash/splash.tsx:29]: I would add a margin mt-4 to seperate the text blocks a bit more for mobile

File: src/components/splash/splash.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+					>
+						<CloseIcon />
+					</button>
+					<div className="flex lg:hidden">
+						<LanguageToggle />
+					</div>
+				</div>
+				<div className="flex flex-row w-full justify-between items-top pb-8">
+					<div className="flex flex-col gap-2 pr-6 w-full lg:max-w-[70%]">
+						<Markdown className="xl:text-2xl 2xl:text-3xl font-semibold">
```
Comment (aeschi) [inline src/components/splash/splash.tsx:73]: here I would add another step for sm:text-xl

**Review by aeschi — CHANGES_REQUESTED** [review]
already looks super good!

a few comments:
- I changed the size of the language toggle earlier but it's not in this branch:
the className of the button inside LanguageToggle should be:
className="bg-gdk-white h-[51px] justify-center items-center hover:text-gdk-light-gray text-gdk-gray flex"

after changing this the padding for the language button on the splash screen on mobile should be correct

**Comment by Jaszkowic** [comment]
@aeschi merged latest feat/ui-rework and fixed your suggestions

**Review by aeschi — APPROVED** [review]
ship it! 🚢

**Comment by aeschi** [comment]
one more thing: we could exchange the <a> tags after this PR #780

**Comment by aeschi** [comment]
went through these comments:

- we seem to have lost the tree icon of the logo in the new design. I'd opt for adding this back in as we're using this on https://deinestadt.giessdenkiez.de/ and in our email templates. 
- mini typo in banner for new landing page: Gieß den Kiez auch in deiner Deiner Stadt
- for EN version please use Connect for last block instead of Network
- link "Erfahre mehr" on mobile threw blank page first time around

File: src/components/router/router.tsx [hunk 4]
```diff
@@ -22,6 +23,8 @@ export const Router: React.FC = () => {
 		store.setIsFilterViewVisible,
 	]);
 
+	const [showSplashScreen, setShowSplashScreen] = useState(true);
```
Comment (raphael-arce) [inline src/components/router/router.tsx:26]: could we make this `isSplashScreenVisible` ?

**Review by raphael-arce — APPROVED** [review]
one tiny suggestion, otherwise lgtm 👍

**Pull request merged**